packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -31,10 +32,10 @@ const Guide = ({ connector, onClose }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { id: connectorId, type: connectorType, name, configTemplate, readme } = connector;
 
-  const locale = i18next.language;
+  const localeRaw = i18next.language;
+  const locale = languageEnumGuard.parse(localeRaw);
```

Comment:
will fallback to?

Reply:
we set 'en' as fallbackLng when initializing i18next, so I think i18next.language will be 'en' if not specified?

Reply:
`parse` will throw an error?

Reply:
I thought that if `localRaw` is not in the `name`'s keys, it is informative enough to throw ZodError to show that `localRaw` is currently not supported?
Or should we throw an error in some other error types?

Reply:
I think this should fallback to `en`, instead of killing the app.

Reply:
Besides, this is a refactor PR, is not expected to change the logic.

Reply:
sounds reasonable, will fix this soon.

Reply:
Updated, PTAL